events-monitor/main.go

```diff
@@ -64,6 +64,18 @@ func main() {
 				Value:       "http://localhost:3500",
 				Usage:       "HTTP standard API endpoint for an Ethereum beacon node",
 			},
+			&cli.DurationFlag{
+				Name:        "store-dumps-interval",
+				Destination: &monitorFlags.storeDumpsInterval,
+				Value:       time.Minute * 5,
+				Usage:       "Interval to store forkchoice dumps (default 5m)",
+			},
+			&cli.DurationFlag{
+				Name:        "purge-dumps-after",
+				Destination: &monitorFlags.purgeDumpsInterval,
+				Value:       time.Hour * 48,
+				Usage:       "Interval to purge forkchoice dumps (default 48h)",
+			},
```

Comment:
This isn't implemented anywhere, but we can rely on gcp bucket object TTL. Want to delete the flag for now?

```suggestion
```